etc/concourse/scripts/cf-deploy-build

```diff
@@ -10,6 +10,10 @@ cp -r landscape/cf-abacus/. built-project
 
 abacus_config_dir="landscape/abacus-config/deploy"
 
+if [[ -e bin/pre-build ]]; then
+  ./bin/pre-build $abacus_config_dir
```

Comment:
I think this should be `pre-configure` and executed only if we want to replace templates